cg/apps/loqus.py

```diff
@@ -25,40 +25,37 @@ class LoqusdbAPI():
     def __init__(self, config: dict):
         super(LoqusdbAPI, self).__init__()
 
-        # For loqusdb v1.0 (Does not take an uri)
-        self.password = config['loqusdb']['password']
-        self.username = config['loqusdb']['username']
-        self.port = config['loqusdb']['port']
-        self.host = config['loqusdb'].get('host') or 'localhost'
-
+        self.uri = config['loqusdb']['database']
         self.db_name = config['loqusdb']['database_name']
         self.loqusdb_binary = config['loqusdb']['binary']
+
         # This will allways be the base of the loqusdb call
         self.base_call = [self.loqusdb_binary, '-db', self.db_name,
```

Comment:
I think it would be awesome to do as sugested in #247 and give the loqus-bin the loqus-config instead of the loqus-configs specifics, what do you think @adrosenbaum ?

Reply:
I think that is a nice idea! I'll fix that.